siriuspy/siriuspy/devices/bpm.py

```diff
@@ -746,139 +627,134 @@ def acq_channel_str(self):
+        return self['GENTriggerDataSel-RB']
 
     @acq_trig_datasel.setter
     def acq_trig_datasel(self, val):
         """."""
-        self['ACQTriggerDataSel-SP'] = val
+        self['GENTriggerDataSel-SP'] = val
 
     @property
     def acq_trig_datathres(self):
         """."""
-        return self['ACQTriggerDataThres-RB']
+        return self['GENTriggerDataThres-RB']
 
     @acq_trig_datathres.setter
     def acq_trig_datathres(self, val):
         """."""
-        self['ACQTriggerDataThres-SP'] = val
+        self['GENTriggerDataThres-SP'] = val
 
     @property
     def acq_trig_datahyst(self):
         """."""
-        return self['ACQTriggerDataHyst-RB']
+        return self['GENTriggerDataHyst-RB']
 
     @acq_trig_datahyst.setter
     def acq_trig_datahyst(self, val):
         """."""
-        self['ACQTriggerDataHyst-SP'] = val
+        self['GENTriggerDataHyst-SP'] = val
 
     @property
     def acq_trig_datapol(self):
         """."""
-        return self['ACQTriggerDataPol-RB']
+        return self['GENTriggerDataPol-RB']
 
     @acq_trig_datapol.setter
     def acq_trig_datapol(self, val):
         """."""
-        self['ACQTriggerDataPol-SP'] = val
+        self['GENTriggerDataPol-SP'] = val
 
     @property
     def acq_nrsamples_post(self):
         """."""
-        return self['ACQSamplesPost-RB']
+        return self['GENSamplesPost-RB']
 
     @acq_nrsamples_post.setter
     def acq_nrsamples_post(self, val):
         """."""
-        self['ACQSamplesPost-SP'] = val
+        self['GENSamplesPost-SP'] = val
 
     @property
     def acq_nrsamples_pre(self):
         """."""
-        return self['ACQSamplesPre-RB']
+        return self['GENSamplesPre-RB']
 
     @acq_nrsamples_pre.setter
     def acq_nrsamples_pre(self, val):
         """."""
-        self['ACQSamplesPre-SP'] = val
+        self['GENSamplesPre-SP'] = val
 
     @property
     def acq_nrshots(self):
         """."""
-        return self['ACQShots-RB']
+        return self['GENShots-RB']
 
     @acq_nrshots.setter
     def acq_nrshots(self, val):
         """."""
-        self['ACQShots-SP'] = val
+        self['GENShots-SP'] = val
 
     def wait_acq_finish(self, timeout=10):
         """Wait Acquisition to finish."""
         return self._wait(
-            'ACQStatus-Sts', self.ACQSTATES_FINISHED, timeout=timeout,
+            'GENStatus-Mon', self.ACQSTATES_FINISHED, timeout=timeout,
             comp=lambda x, y: x in y)
 
     def wait_acq_start(self, timeout=10):
         """Wait Acquisition to start."""
         return self._wait(
-            'ACQStatus-Sts', self.ACQSTATES_STARTED, timeout=timeout,
+            'GENStatus-Mon', self.ACQSTATES_STARTED, timeout=timeout,
             comp=lambda x, y: x in y)
 
     def cmd_acq_start(self):
         """Command Start Acquisition."""
         self.acq_ctrl = _csbpm.AcqEvents.Start
-        return self._wait('ACQTriggerEvent-Sts', _csbpm.AcqEvents.Start)
+        return True
```

Comment:
Isn't there a way to know if the command was implemented anymore?

Reply:
As we talked offline, we don't have a PV for that now, only the check for GENStatus-Mon value, as wait_acq_finish already does